pkg/certgenerator/v1beta1/generator.go

```diff
@@ -67,11 +70,34 @@ func (c *CertGenerator) Start(ctx context.Context) error {
 	if err := c.generate(ctx); err != nil {
 		return err
 	}
+	if err := wait.ExponentialBackoffWithContext(ctx, wait.Backoff{
```

Comment:
Is there a requirement for a timeout? How does the user get to know that it is being waited for certs to be ready?

Reply:
> Is there a requirement for a timeout?

Yes, timeout is 5 minutes.

> How does the user get to know that it is being waited for certs to be ready?

The controller pod doesn't ready until the cert is ready.

```shell
NAME                                READY   STATUS              RESTARTS   AGE
katib-controller-687798c75b-mtlz7   0/1     Running             0          4s

status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2023-08-13T11:48:12Z"
    message: 'containers with unready status: [katib-controller]'
    reason: ContainersNotReady
    status: "False"
    type: ContainersReady
controller@sha256:86697de94ac8dfbe53082beb96f1ea4ffea2bac982a2332719a99224e17335c7
    lastState: {}
    name: katib-controller
    ready: false
    restartCount: 0
    started: true
    state:
      running:
        startedAt: "2023-08-13T11:48:14Z"
```

Reply:
I meant, how will the user know that Katib controller is waiting for certs to get ready?

Reply:
The controller doesn't say, `Waiting for certs to get ready`. So, there are two ways that users can indirectly know whether the controller waits for certs to get ready:

1. As shown above, confirmation pod condition.
2. Creating the Experiments. If the controller waits for certs, users can get the following error:

```
Error from server (InternalError): error when creating "../../testdata/valid-experiment.yaml": Internal error occurred: failed calling webhook "defaulter.experiment.katib.kubeflow.org": could not get REST client: unable to load root certificates: unable to parse bytes as PEM block
```

Adding a log, `Waiting for certs to get ready`, to L72 in the file might be better.

Reply:
Adding a log is better because user can understand the reason behind the issue

Reply:
Added.